### ISSUE TITLE
Bust reconstruction

### ISSUE THREAD

**Issue by Vanessik** [GAP-LAB-CUHK-SZ/HairStep#24]
How to obtain the bust aligned with hair reconstructions?

**Comment by Zvyozdo4ka** [comment]
> How to obtain the bust aligned with hair reconstructions?

the bust can be generated separately, or use their head model from [`data` folder. ](https://github.com/GAP-LAB-CUHK-SZ/HairStep/blob/main/data/head_model.obj)

**Comment by Vanessik** [comment]
@Zvyozdo4ka yes, i got it, thanks, but did you understand which method was used for bust generation?

**Comment by Zvyozdo4ka** [comment]
> @Zvyozdo4ka yes, i got it, thanks, but did you understand which method was used for bust generation?

the output of generation only hair strands, they do not mention bust generation and the project does not generate head
